clients/itwin/src/Request.ts

```diff
@@ -337,10 +337,10 @@ export async function request(requestContext: ClientRequestContext, url: string,
   if (options.parser)
     sareq = sareq.parse(options.parser);
 
-  if (options.agent)
-    sareq = sareq.agent(options.agent);
-  else if (RequestGlobalOptions.httpsProxy)
+  if (RequestGlobalOptions.httpsProxy)
```

Comment:
Won't this prevent the iModelHub team's agent settings to be used when using Fiddler or any other proxy?

On a related note, I am working on a set of changes to add proxy support to all the API including the iModelHub API - that'll avoid the need to swap this around. Please undo this change for now - fiddler/proxy support will be restored after my changes.